src/triggers/errors.ts

```diff
@@ -7,7 +7,7 @@ export const errorRetry: Trigger<undefined> = async (clients) => {
   const errorQuery = `select *
   from "${Table.nftProcessErrors}"
   where ("numberOfRetries" < '${NUMBER_OF_RETRIES}' or "numberOfRetries" is null)
-  and (extract(minute from '${new Date().toISOString()}' - "lastRetry") >=1 or "lastRetry" is null)
```

Comment:
I swapped `${new Date().toISOString()}` with `now()` so we use the database time. Any pitfalls I'm not seeing with using DB time over the nodejs process's time?

Reply:
should be fine if it's the same type/format